recipes/cf-nvidia-tools/recipe.yaml

```
@@ -0,0 +1,46 @@
+context:
+  name: cf-nvidia-tools
+  version: "1.0.0"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - path: bin
+    target_directory: bin
+
+build:
+  skip:
+    - not linux
+  noarch: generic
+  script:
+    - ls -lah
+    - cp -r bin $PREFIX/bin
+  number: 0
+
+requirements:
+  run:
+    - __unix
+    - patchelf >=0.12
+
+tests:
+  - script:
+    - export c_stdlib_version="2.17.0"
```

Should we do some kind of help check?

```suggestion
    - check-glibc
    - export c_stdlib_version="2.17.0"
```

---

Done.